pnpm-lock.yaml

```diff
@@ -3314,118 +3314,6 @@ importers:
         specifier: 5.6.2
         version: 5.6.2
 
-  packages/cli:
```

Comment:
Makes the necessary symlinking changes as part of moving `packages/cli` -> `package/novu`, plus a few minor dependency updates.